scripts/templates/create-package/EmptyPackageJson.mustache

```diff
@@ -33,21 +36,19 @@
     "@types/react-test-renderer": "{{{typesReactTestRenderer}}}",
     "@types/webpack-env": "{{{typesWebpackEnv}}}",
     "@uifabric/build": "{{{uifabricBuild}}}",
-    "@uifabric/example-app-base": "{{{uifabricExampleAppBase}}}",
-    "@uifabric/jest-serializer-merge-styles": "{{{uifabricJestSerializerMergeStyles}}}",
     "@uifabric/tslint-rules": "{{{uifabricTslintRules}}}",
     "enzyme": "{{{enzyme}}}",
     "enzyme-adapter-react-16": "{{{enzymeAdapterReact16}}}",
     "es6-weak-map": "{{{es6WeakMap}}}",
     "react": "{{{react}}}",
     "react-app-polyfill": "{{{reactAppPolyfill}}}",
     "react-dom": "{{{reactDom}}}",
-    "react-test-renderer": "{{{reactTestRenderer}}}"
+    "react-test-renderer": "{{{reactTestRenderer}}}",
+    "typescript-plugin-css-modules": "{{{typescriptPluginCssModules}}}"
+
   },
   "dependencies": {
-    "@microsoft/load-themed-styles": "{{{microsoftLoadThemedStyles}}}",
     "@uifabric/set-version": "{{{uifabricSetVersion}}}",
```

Comment:
Do we need this?

Reply:
This was added by ken to detect which package versions are used in an application.